Doc/library/stdtypes.rst

```diff
@@ -39,26 +39,17 @@ Truth Value Testing
    single: false
 
 Any object can be tested for truth value, for use in an :keyword:`if` or
-:keyword:`while` condition or as operand of the Boolean operations below. The
-following values are considered false:
+:keyword:`while` condition or as operand of the Boolean operations below.
 
   .. index:: single: None (Built-in object)
-
-* ``None``
-
   .. index:: single: False (Built-in object)
 
-* ``False``
-
-* zero of any numeric type, for example, ``0``, ``0.0``, ``0j``.
-
-* any empty sequence, for example, ``''``, ``()``, ``[]``.
-
-* any empty mapping, for example, ``{}``.
-
-* instances of user-defined classes, if the class defines a :meth:`__bool__` or
-  :meth:`__len__` method, when that method returns the integer zero or
-  :class:`bool` value ``False``. [1]_
+Objects are considered false if a :meth:`__bool__` or :meth:`__len__` method is
+defined on them, when that method returns the integer zero or :class:`bool`
+value ``False``. [1]_  Practically, this means that empty containers (such as
+``[]``, ``()``, ``{}``, ``''``, etc.) are false and that numbers equal to zero
```

Comment:
@peterthomassen just FYI, 'collection' is technically correct here and 'container' is technically wrong, since the Container abc only requires __contains__ to be defined, which if nothing else was defined would make the object true.  A Collection, on the other hand, is sized, and so can be false if empty.

You are right, however, that we do sometimes use them interchangeably.  But we probably shouldn't :)

Thanks for taking the time to work through this PR.